src/commands/src/helpers/dependency-helpers.ts

```diff
@@ -43,16 +43,25 @@ async function createEnvProdFile({
   }
 }
 
-export async function downloadBuildDependencies(buildDependencies: BuildDependencies, repoName: string) {
+export async function downloadBuildDependencies(
+  buildDependencies: BuildDependencies,
+  repoName: string,
+  directory?: string
+) {
   const commands: string[] = [];
   await Promise.all(
     buildDependencies.map(async (dependencyInfo) => {
-      const repoDir = getRepoDir(repoName);
+      const repoDir = directory ? getRepoDir(repoName, directory) : getRepoDir(repoName);
```

Comment:
No need to do this conditional! The `directory` argument can be undefined. You can simple state:

`const repoDir = getRepoDir(repoName, directory);`